XenAdmin/SettingsPanels/PerfmonAlertOptionsPage.cs

```diff
@@ -132,8 +138,14 @@ public void Repopulate()
                     EmailAddressTextBox.Text = _PerfmonOptions.MailDestination;
                     SmtpServerAddrTextBox.Text = PerfmonOptionsDefinition.GetSmtpServerAddress(_PerfmonOptions.MailHub);
                     SmtpServerPortTextBox.Text = PerfmonOptionsDefinition.GetSmtpPort(_PerfmonOptions.MailHub);
+                    if (null != _PerfmonOptions.MailLanguageCode)

```

Comment:
MailLanguageCode always returns null because the language selection is not saved if changed from XenCenter. It needs to be added to PerfmonOptionsDefinitionAction in order to be saved to the pool's other-config.